### ISSUE TITLE
No need for these pie charts

### ISSUE THREAD

**Issue by CWWhitney** [AlexandraKrause/Thesis#1]
https://github.com/AlexandraKrause/Thesis/blob/3f6c0c9a4e71d4777a4a71f4f8bd709b9c36dbc0/Topics_pie_chart.R#L52

**Comment by AlexandraKrause** [comment]
oh i see. i just wanted to compare it to this one from ciar
[image: image.png]


Am Mi., 22. Juni 2022 um 15:06 Uhr schrieb Cory Whitney <
***@***.***>:

**Comment by CWWhitney** [comment]
Ok. I see the use of the once char - maybe reference the CGIAR chart for comparison and remove the rest of the pie charts?

**Comment by AlexandraKrause** [comment]
I made these pie charts per topic because you commented on my original
expose: "The pathway will indicate which methods were used within the
studies that were the knowledge basis for the relationships described
(i.e., the conceptual model will reflect the findings of the literature in
the form of a qualitative model)." I saw this note as an indication that I
should create them and we also discussed this before I made them. Only,
therefore, I created this big excel table. Are you sure I should put them
out? I can do this, but I want to make sure.

Best wishes,
Alexandra

Am Mi., 22. Juni 2022 um 22:07 Uhr schrieb Cory Whitney <
***@***.***>:
